pkg/targetgroupbinding/targets_manager.go

```diff
@@ -296,3 +313,27 @@ func pointerizeTargetDescriptions(targets []elbv2sdk.TargetDescription) []*elbv2
 func cloneTargetInfoSlice(targets []TargetInfo) []TargetInfo {
 	return append(targets[:0:0], targets...)
 }
+
+func (m *cachedTargetsManager) isInCIDRRange(ipAddr string) (bool, error) {
```

Comment:
turbo nit: idk why but this helper feels like it probably doesn't need to be a method on `cachedTargetsManager`, and is a better fit for a func on it's own

like `isInCIDRRange(cidrBlocks, ipAddr)`.

Change if u agree, keep it if u don't care